cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/async/progress/OperationProgress.java

```diff
@@ -115,4 +129,38 @@ private void ensureMutable() {
       throw new IllegalStateException("Cannot change this operation progress because it is immutable.");
     }
   }
+
+  @JsonResponseClass
+  private static class StepProgress {
+    @JsonResponseField
+    static final String STEP = "step";
+    @JsonResponseField
+    static final String DESCRIPTION = "description";
+    @JsonResponseField
+    static final String TIME_IN_MS = "timeInMs";
```

Comment:
This patch seems to introduce backwards incompatible name changes, which potentially breaks the existing clients / UIs that depend on CC.

Can we ensure that no such backwards incompatible changes are introduced?

Reply:
Sure, I will revert them back.